package.json

```diff
@@ -28,8 +28,8 @@
     "docker-build": "yarn tsc && yarn workspace example-backend build && yarn workspace example-backend build-image",
     "fix": "backstage-cli repo fix --publish",
     "postinstall": "husky || true",
-    "lint": "backstage-cli repo lint --since origin/master",
-    "lint:all": "backstage-cli repo lint",
+    "lint": "backstage-cli repo lint --since origin/master --quiet",
```

Comment:
I'm not sure if this is necessary. Everything should work even without this change, because earlier every `warn` was error.